hack/source-manifests.sh

```diff
@@ -80,7 +80,7 @@ gen_args="olm-catalog gen-csv --csv-version=$TMP_CSV_VERSION"
 if [ -n "$CSV_REPLACES_VERSION" ]; then
 	gen_args="$gen_args --from-version=$CSV_REPLACES_VERSION"
 fi
-$OPERATOR_SDK "$gen_args" 2>/dev/null
+$OPERATOR_SDK $gen_args 2>/dev/null
```

Comment:
Can we also remove the `2>/dev/null` and put `echo "Generating intermediate csv: Ignore warnings about missing fields spec.keywords, spec.maintainers, spec.provider"` before the line so it's not necessary?

Reply:
Agree with you @rohantmp ... We could even exit (rather than continuing) if the command fails...
@kshlm thoughts?

Reply:
`set -e` at the beginning of the script already ensures that any failed commands will cause the script to exit :)